data/transactions/logic/README_in.md

```diff
@@ -193,6 +193,41 @@ _available_.
 
  * Since v7, the account associated with any contract present in the
    `txn.ForeignApplications` field is _available_.
+   
+ * Since v9, there is group-level resource sharing. Any resource that
+   is available in _some_ top-level transaction in a transaction group
+   is available in _all_ v9 or later application calls in the group,
+   whether those application calls are top-level or inner.
+   
+ * When considering whether an asset holding or application local
+   state is available by group-level resource sharing, the holding or
+   local state must be available in a top-level transaction without
+   considering group sharing. For example, if account A is made
+   available in one transaction, and asset X is made available in
+   another, group resource sharing does _not_ make A's X holding
+   available.
+     
+ * Top-level transactions that are not application calls also make
+   resources available to group-level resource sharing. The following
+   resources are made available by other transaction types.
+
+     1. `pay` - `txn.Sender`, `txn.Receiver`, and
```

Comment:
This is the right way to do numbering in markdown. Generated html puts in the current numbers. By using 1. everywhere, diffs can be smaller in the future.

The app transaction rules are described in the previous paragraph because they are quite different. With the non-app calls, it's easy to say "here's the list". With apps, we describe it more inductively. We're saying "There are a bunch of rules about availability for apps already, v9 makes those rules apply to all txns."  The list is needed for non-apps because they had no previous rules for availability.

```
 * Since v9, there is group-level resource sharing. Any resource that
   is available in _some_ top-level transaction in a transaction group
   is available in _all_ v9 or later application calls in the group,
   whether those application calls are top-level or inner.
   
 * When considering whether an asset holding or application local
   state is available by group-level resource sharing, the holding or
   local state must be available in a top-level transaction without
   considering group sharing. For example, if account A is made
   available in one transaction, and asset X is made available in
   another, group resource sharing does _not_ make A's X holding
   available.
```